tests/providers/openlineage/plugins/test_listener.py

```diff
@@ -210,7 +210,7 @@ def mock_task_id(dag_id, task_id, try_number, execution_date):
     task_instance.state = State.RUNNING
     task_instance.start_date = dt.datetime(2023, 1, 1, 13, 1, 1)
     task_instance.end_date = dt.datetime(2023, 1, 3, 13, 1, 1)
-    task_instance.execution_date = "execution_date"
+    task_instance.execution_date = "logical_date"
```

Comment:
It should be a certain date, and not just a string, I think. Maybe we can fix it here.

Reply:
Fixed.